README.md

```diff
@@ -1,6 +1,58 @@
-# tds-community
+# TELUS Design System Community
 
-TDS community components
+The TELUS Design System (TDS) is a set of living guidelines that communicates our brand promise through our digital experiences.
+It's a holistic platform that integrates documentation, guidelines and design management that serves as single source for
+digital design guidelines, code patterns and UI elements.
+
+## Introduction
+
+The TDS Community is a collaborative platform where TELUS designers and developers can rapidly discover, share, and consume
+their modules as part of the TELUS Design System. This includes React components, the [component library for designers](https://tds.telus.com/getting-started/designers.html), and their documentation.
+
+TDS Community is similar to [TDS Core](https://github.com/telusdigital/tds-core) since community components
+have the same technical and branding foundations as core components, however the underlying governance model and
+maintenance workflow differs. Here is a high level summary:
+
+| Feature               | TDS Core      | TDS Community                |
+| --------------------- | ------------- | ---------------------------- |
+| Component Reusability | High          | Moderate                     |
+| Release frequency     | Low           | High                         |
+| Contribution speed    | Moderate      | High                         |
+| Review board          | TDS Core Team | Digital Platform Ambassadors |
+
+The [Digital Platform Ambassadors](https://github.com/orgs/telusdigital/teams/digital-platform-ambassadors) are a team
+of representatives from every tribe at TELUS digital. They help grow the digital platform and will actively
+review contributions for the TDS Community to ensure quality standards are met.
+
+## Getting started
+
+Consuming TDS Community Sketch assets or React components is very similar to TDS Core. You can follow our
+[getting started](https://tds.telus.com/getting-started/getting-started.html) guide to learn more.
+
+### For designers
+
+During the initial launch of TDS Community, component designs will be audited by internal TELUS design
+leads and will be delegated to their respective tribe developers for contribution. These notes will be updated
+once an appropriate place for supplementary Sketch assets are decided.
+
+### For developers
+
+Unlike TDS Core, npm modules from TDS Community are prefixed `@tds/community-*`. To set up your project
+to consume community or core components:
+
+1.  Follow the [getting started guide for developers](https://tds.telus.com/getting-started/developers.html)
+2.  Go to the [TDS Community catalogue](https://tds.telus.com/community/index.html) and choose a component to install
+
+## Contributing
+
+To learn how to make contributions to TDS Community, See the [contributing guide](https://tds.telus.com/contributing/contributing.html).
+
+## Roadmap
```

Comment:
I'd probably take this out for now as this section really doesn't seem to give any info. Perhaps the README could somehow indicate that TDS Community is in "Beta" right now though. Maybe at the top.